rfc/rfc015-node-identity.md

```diff
@@ -0,0 +1,126 @@
+# RFC015 Node identity
+
+|  |  |
+| :--- | :--- |
+| Nuts foundation | W.M. Slakhorst |
+| Request for Comments: 015 | Nedap |
+|  | Nov 2021 |
+
+## Node identity
+
+### Abstract
+
+This RFC describes how to identify a node in the network. Node identification is required for exchanging private data.
+Measures to keep data private can range from encrypting data to sending data over a specific connection.
+The TLS connection and certificate exchange form the basis of the node identification.
+The *Subject Alternative Name* of the certificate is matched against the *serviceEndpoint* of a DID Document service.
+The requirement for the additional service in the DID Document also makes it possible to find other nodes and thus enable service discovery of Nuts nodes.
+
+Private data, like authorization credentials, have DIDs as issuer and subject. To enable private addressing, a DID will be encrypted with its public encryption key.
+Only the owner of the private key can decrypt this. Doing this on the individual DID level will be very inefficient.
+Each node would have to try all the private keys it manages to find out if a piece of data is addressed to it.
+To make this more efficient this RFC links individual DIDs to the service provider DID.
+An encrypted address would then only have to be decrypted with the service provider DID.
+
+### Status
+
+This document is currently a draft.
+
+### Copyright Notice
+
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1. Introduction
+
+When all participants of a network share the same public data, there's no need to know which node services a particular organization.
+This changes when data becomes private. Private data needs to be routed to the correct node.
+In order for that to work, there's a need to identify each node and link that identity to services organizations.
+[RFC014](rfc014-authorization-credential.md) describes that Nuts authorization credentials need to remain private.
```

Comment:
The last sentence feels a little bit 'off' here. Maybe move it to a section that explains more about private data?

Reply:
I removed that sentence